configure.ac

```diff
@@ -133,7 +134,6 @@ AS_CASE([$host],
 
     # externals are dynamically linked to pd.dll in their individual automake files
     EXTERNAL_CFLAGS="-mms-bitfields"
-    EXTERNAL_LDFLAGS="-s -Wl,--enable-auto-import -no-undefined -lpd"
```

Comment:
hmmm... wouldn't it make sense to keep the Windows specific flags here and then just append `-s` later in the `AS_IF([test "x${debug}"` clause?

Also, `-s` should be added to a debug build for *all* platforms, not only on Windows. (In practice, the default CFLAGS may already contain the `-s` flag, but it wouldn't hurt to set it explicitly. After all, that's what you want when you ask for a debug build...)